packages/markup-frontstage/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+module.exports = {
```

Comment:
any reason the jest config cant be moved back into the package.json? or move the rest of the jest config in here?

Reply:
Yes.  The reason that the tests were failing is because CRA only supports a small subset of jest configuration options in the package.json file.  If you include jest.config.js, you can include the `moduleNameMapper` property, which is necessary in order to use @import in the ui-core scss files.  Previously `moduleNameMapper` was in the package.json file, but it didn't do anything at all.